--- kinetic_piqi.ml	2014-09-10 16:29:27.571353723 +0200
+++ kinetic_piqi.new.ml	2014-09-10 16:29:19.031353401 +0200
@@ -236,7 +236,7 @@
 and Message_get_log:
   sig
     type t = {
-      mutable type: Kinetic_piqi.message_get_log_type list;
+      mutable _type: Kinetic_piqi.message_get_log_type list;
       mutable utilization: Kinetic_piqi.message_get_log_utilization list;
       mutable temperature: Kinetic_piqi.message_get_log_temperature list;
       mutable capacity: Kinetic_piqi.message_get_log_capacity option;
@@ -602,7 +602,7 @@
   let _limits, x = Piqirun.parse_optional_field 8 parse_message_get_log_limits x in
   Piqirun.check_unparsed_fields x;
   {
-    Message_get_log.type = _type;
+    Message_get_log._type = _type;
     Message_get_log.utilization = _utilization;
     Message_get_log.temperature = _temperature;
     Message_get_log.capacity = _capacity;
@@ -1080,7 +1080,7 @@
   Piqirun.gen_record code (_hostname :: _port :: _tls :: [])
 
 and gen__message_get_log code x =
-  let _type = Piqirun.gen_repeated_field 1 gen__message_get_log_type x.Message_get_log.type in
+  let _type = Piqirun.gen_repeated_field 1 gen__message_get_log_type x.Message_get_log._type in
   let _utilization = Piqirun.gen_repeated_field 2 gen__message_get_log_utilization x.Message_get_log.utilization in
   let _temperature = Piqirun.gen_repeated_field 3 gen__message_get_log_temperature x.Message_get_log.temperature in
   let _capacity = Piqirun.gen_optional_field 4 gen__message_get_log_capacity x.Message_get_log.capacity in
@@ -1463,7 +1463,7 @@
   }
 and default_message_get_log () =
   {
-    Message_get_log.type = [];
+    Message_get_log._type = [];
     Message_get_log.utilization = [];
     Message_get_log.temperature = [];
     Message_get_log.capacity = None;
